### PR TITLE
Shortened cloudant url by removing username:password from the URL

### PR DESCRIPTION
**Pull request by sanjeevghimire**

### CODE REVIEW

**Review by scottdangelo — CHANGES_REQUESTED** [review]
Looks fine, but needs a commit message.

**Comment by scottdangelo** [comment]
Looks like flake8 failures in CI:

Collecting flake8
  Downloading flake8-3.5.0-py2.py3-none-any.whl (69kB)
    100% |████████████████████████████████| 71kB 5.7MB/s 
Collecting pyflakes<1.7.0,>=1.5.0 (from flake8)
  Downloading pyflakes-1.6.0-py2.py3-none-any.whl (227kB)
    100% |██���█████████████████████████████| 235kB 4.8MB/s 
Requirement already satisfied: enum34; python_version < "3.4" in /home/travis/virtualenv/python2.7.14/lib/python2.7/site-packages (from flake8)
Collecting configparser; python_version < "3.2" (from flake8)
  Downloading configparser-3.5.0.tar.gz
Collecting pycodestyle<2.4.0,>=2.0.0 (from flake8)
  Downloading pycodestyle-2.3.1-py2.py3-none-any.whl (45kB)
    100% |████████████████████████████████| 51kB 7.1MB/s 
Collecting mccabe<0.7.0,>=0.6.0 (from flake8)
  Downloading mccabe-0.6.1-py2.py3-none-any.whl
Building wheels for collected packages: configparser
  Running setup.py bdist_wheel for configparser ... -done
  Stored in directory: /home/travis/.cache/pip/wheels/1c/bd/b4/277af3f6c40645661b4cd1c21df26aca0f2e1e9714a1d4cda8
Successfully built configparser
Installing collected packages: pyflakes, configparser, pycodestyle, mccabe, flake8
Successfully installed configparser-3.5.0 flake8-3.5.0 mccabe-0.6.1 pycodestyle-2.3.1 pyflakes-1.6.0
0.95s$ flake8 .
./run.py:148:80: E501 line too long (92 > 79 characters)
./watsononlinestore/database/cloudant_online_store.py:188:5: E303 too many blank lines (2)
./watsononlinestore/database/cloudant_online_store.py:189:80: E501 line too long (81 > 79 characters)
./watsononlinestore/database/cloudant_online_store.py:193:15: E225 missing whitespace around operator
./watsononlinestore/database/cloudant_online_store.py:204:1: W391 blank line at end of file

**Comment by stevemar** [comment]
@sanjeevghimire something like this should work:

```
        # Instantiate Cloudant DB.
        safe_url = CloudantOnlineStore.make_cloudant_url_compatible_with_py3(cloudant_url),
        cloudant_online_store = CloudantOnlineStore(
            Cloudant(
                cloudant_username,
                cloudant_password,
                url=safe_url,
                connect=True
            ),
            cloudant_db_name
        )
```

might have to shorten the method name.

**Review by scottdangelo — APPROVED** [review]

File: ?
Comment (stevemar) [inline ?:1]: super minor optimization, you *could* only run this if it's py3 by adding a guard like:

```
import sys
if sys.version_info[0] < 3:
    return url
...
# rest of the code
```

**Review by stevemar — APPROVED** [review]
LGTM

**Pull request merged**